icacls_work.rb

```diff
@@ -0,0 +1,36 @@
+class Rights
+
+  attr_reader :path, :name
+  def initialize(path, name)
+    @path = path
+    @name = name
+  end
+
+  def get_res_string(name_file)
+    system("cd #{@path} && icacls #{@name} > #{name_file}.txt")
+    written_file = File.open("#{@path}\\#{name_file}.txt")
+    written_file.each do |line|
+      return line if line.include?'Users'
+    end
+  end
+
+  def print_rights(rights_line)
+    res_string = 'Users have '
+    case
+    when rights_line.include?('(F)') then res_string += 'full access'
```

Comment:
Create a hash instead of case-when block. Look for the symbol in the method and write a line according to it.

Reply:
Done. Use hash 'permission' for working